lib/DDG/Spice/Forecast.pm

```diff
@@ -22,7 +22,15 @@ handle remainder => sub {
     return if $_;
 
     my $loc_str = join " ", map { $loc->{$_} } @locs;
-    return $loc_str, 'current', {is_cached => 0};
+
+    # Default to English
+    my $parsed_lang = 'en';
+    # And try to extract language from locale
+    if ($lang && $lang->locale) {
+        ($parsed_lang) = $lang->locale =~ /_([A-Z]{2})$/;
```

Comment:
It's failing on all weather requests for me because it's sending lang to forecast as 'us'